ast/desugar/Desugar.cc

```diff
@@ -1619,10 +1619,22 @@ unique_ptr<Expression> node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Nod
                 result.swap(res);
             },
             [&](parser::Defined *defined) {
+                auto valuePtr = node2TreeImpl(dctx, std::move(defined->value));
+                auto loc = valuePtr->loc;
+                auto value = valuePtr.get();
+                Array::ENTRY_store args;
```

Comment:
can we pass them as inline args rather than passing an array?

Reply:
Why is that better?